### PR TITLE
307 contrib docs

### PR DESCRIPTION
**Pull request by robpre**
Fixes #

## Description

-
-

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+## Getting tickets/issues ready to be worked on
+
+Before a thing can be worked on it must be both created as an issue, and prioritised using a definition of done.
+
+### Creating issues
+
+1. Open the `Issues` tab and click [`New issue`](https://github.com/neontribe/www/issues/new/choose) and choose the right template.
+   - anyone can create issues
+   - `Bug` is for bugs
+   - `Story` is for new things, e.g. code, content, dependency updates
```
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
   - `Story` is for new things, e.g. code, content to meet a user need
   -  `Tasks` is for simple actions such as dependency updates
```

Comment (katquatermass) [reply CONTRIBUTING.md]: The reason for the edit is that when I was working on this with Rob, I misunderstood the old "task" card and turned it into the "Research" card - I think on reflection we need both.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Any issues in the `Top priority issues` column are important enough and have been prioritised.
+
+When contributing to this project please do the following:
+
+1. Pick something near the top of the `Top priority issues` column, which you feel you have the time and confidence to tackle.
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+1. Check the [Definition of done](#definition-of-done) for the ticket to see if you understand it.
+   - If you need more information please follow up with the person who spawned/created the task.
+1. Assign yourself and move the ticket to the `In progress` column.
+1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. _n.b._ [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
```
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. 
- [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
```

Comment (katquatermass) [reply CONTRIBUTING.md]: The reason for the edit is we are developing a pattern whereby the main numbered bullet is the step in the guide, and secondary bullets are either a step that only occurs in certain circumstances OR an encouragement or piece of advice (the encouragement or advice also becomes a third bullet, if they belong to an "if" instruction). Removing the nb's brings these sections in line with the content pattern earlier.

Comment (katquatermass) [reply CONTRIBUTING.md]: My amend is wrong - I haven't got line 56 markdown right and don't know how to change it now!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+1. Work on the feature branch, make small commits which include the _reason_ for each commit's change.
+   - We don't expect you to battle the problem alone, put any queries on the github issue and slack, people will help when they have time.
+1. When you're happy with the changeset/the commits you've made, and you're happy that the ticket passes its [Definition of done](#definition-of-done) create a pull request.
+   _n.b._ Pull requests are the pull requester's responsibility, when it receives the request number
```
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
     

- Pull requests are the pull requester's responsibility, when it receives the request number

```

Comment (katquatermass) [reply CONTRIBUTING.md]: As above, but not sure why the bullet format is not displaying red

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+1. Await code review, see [Review proces](#review-process)
+
+Please make sure that you keep your activity up-to-date:
+
+1. take each ticket as far as you can, before taking another one (e.g. ready for code review)
+   - if the ticket needs to be put on hold because you've reached the limit of your availability or understanding then;
+     - put you findings/thoughts on the ticket
```
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
     - put your findings/thoughts on the ticket
```

Comment (katquatermass) [reply CONTRIBUTING.md]: correct spelling

File: CONTRIBUTING.md (hunk 3 above)
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
   - Once you have made a pull request it is your responsibility to check for reviews. See [review process] (#review-process)

```

Comment (katquatermass) [reply CONTRIBUTING.md]: cleared up meaning as well as changed the bullet format - I think this is right @RpprRoger

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+   - We expect you to learn while tackling issues, please don't be put off by a tasks' seeming complexity.
+1. Check the [Definition of done](#definition-of-done) for the ticket to see if you understand it.
+   - If you need more information please follow up with the person who spawned/created the task.
+1. Assign yourself and move the ticket to the `In progress` column.
+1. Create a branch with a name to help match it to the ticket, e.g.: `307-contrib-docs`. _n.b._ [avoid slashes in branch names](https://stackoverflow.com/questions/2527355/using-the-slash-character-in-git-branch-name/2527452#2527452)
+1. Work on the feature branch, make small commits which include the _reason_ for each commit's change.
+   - We don't expect you to battle the problem alone, put any queries on the github issue and slack, people will help when they have time.
+1. When you're happy with the changeset/the commits you've made, and you're happy that the ticket passes its [Definition of done](#definition-of-done) create a pull request.
+   _n.b._ Pull requests are the pull requester's responsibility, when it receives the request number
+1. Await code review, see [Review proces](#review-process)
```
Comment (katquatermass) [inline CONTRIBUTING.md]:
```suggestion
1. Await code review, see [Review process](#review-process)
```

Comment (katquatermass) [reply CONTRIBUTING.md]: typo edit

**Comment by katquatermass** [comment]
@robpre do we need to action this pull request or do we need to withdraw it and then make a pull request for the work we did last week?

**Comment by zii-67** [comment]
> > Sorry for the late approve, this is a good piece of work as I read. I'm happy to approve it, but I have just 2 questions which is should we add a research option in `new issue` and should we include something about feature?
> 
> Agree - I need to make an issue to update the Task Card and create the Research card. Feature is deliberately removed because our practice is to use Story not Feature in order to make sure a feature is clearly reasoned through. (If people want to disagree with that, then they can disagree on the document and we can go back the other way again - but we shouldn't have both - the Story should only become a Feature as it's plan develops, therefore an issue card for Feature's shouldn't be needed is my logic)

Awesome! Thanks Kat. I am happy to approve it now and you can make a new PR for changing the 'task' card.

**Comment by katjam** [comment]
@katquatermass @robpre @zii-67 Is this one ready for review? Or shall we close until it is?

**Comment by katquatermass** [comment]
Pretty sure it can be reviewed at this stage. There are a couple of related
things (issues cards) that need doing, and potential future documentation
additions/changes (like a new home for the task description of adding a
triber that isn't the readme) but I will do individual subcards for all of
those tomorrow so that this PR can be reviewed as is for now?


*Kat QuatermassService and Strategy Design*
Neontribe
Kat@neontribe.co.uk
0789 1487375

**Pull request merged**